frontend/public/components/cluster-settings/cluster-operator-start-guide.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Button } from 'patternfly-react';
+
+import { ExternalLink } from '../utils';
+
+const seenOperatorStartGuide: string = 'seenOperatorStartGuide';
+
+export class ClusterOperatorStartGuide extends React.Component<ClusterOperatorStartGuideProps, ClusterOperatorStartGuideState> {
+  constructor(props) {
+    super(props);
+
+    this.dismiss = this.dismiss.bind(this);
+    let {visible} = props;
+    try {
+      visible = visible || !localStorage.getItem(seenOperatorStartGuide);
+    } catch (ignored) {
+      // ignored
+    }
+    this.state = {
+      visible,
+    };
+  }
+
+  dismiss() {
+    this.setState({
+      visible: false,
+    });
+    localStorage.setItem(seenOperatorStartGuide, 'true');
+  }
+
+  render() {
+    const { visible } = this.state;
+    return visible
+      ? <div className="co-well">
+        <div className="co-well-header">
+          <Button aria-label="Close" bsStyle="link" onClick={this.dismiss}><i className="pficon pficon-close" aria-hidden="true" /></Button>
+        </div>
+        <h4>What are Cluster Operators?</h4>
+        <p>
+          An Operator is a method of packaging, deploying and managing a Kubernetes application. Cluster Operators implement and automate updates of OpenShift and Kubernetes at the cluster level. During an update, the latest versions of the OpenShift and Kubernetes components are downloaded. A rolling update will occur to install the latest versions. To learn more about how this process is executed, read about <ExternalLink href="https://coreos.com/operators/" text="Operators" />.
```

Comment:
I think the message here could be tightened up. IMHO it doesn't need to be totally comprehensive since we're pointing users to the docs anyway.

> Cluster Operators automate updates of OpenShift and Kubernetes at the cluster level. During an update, the latest component versions downloaded and installed. To learn more about this process, read about Operators.

I don't want to block this PR, but we are going to discuss from a design perspective when using these blue boxes makes sense vs using an infopop, as in #1431.

Reply:
> I don't want to block this PR, but we are going to discuss from a design perspective when using these blue boxes makes sense vs using an infopop, as in #1431.

It would be good to find a pattern we could use on other pages. I think describing what a resource is makes sense for most resources, but the blue box at the top of every page would be too much.

Reply:
Do you still want to change the message given that we can't include the link right now?